### ISSUE TITLE
Issues with the tf-keras-contrib module on conda

### ISSUE THREAD

**Issue by Adithya-MN** [davidADSP/GDL_code#26]
I was trying to set up my conda environment for the TF 2 version of the book, but am not able to install the tf-keras-contrib module It doesn't seem to be available on conda/conda-forge/pip. I'm using Python 3.6.9 for reference.

![Screenshot from 2019-11-04 10-32-46](https://user-images.githubusercontent.com/30416447/68100551-a1b25000-feee-11e9-8b3a-1c569eb2593f.png)

I found [this issue](https://github.com/davidADSP/GDL_code/issues/9) referencing keras-contrib, but unfortunately couldn't find a similiar thread for tf-keras-contrib. Searching for issues related to tf-keras-contrib seems to return results for keras.contrib.

I found [this issue](https://github.com/keras-team/keras-contrib/issues/512) on the official repo that talks about this.

![Screenshot from 2019-11-04 10-12-16](https://user-images.githubusercontent.com/30416447/68100563-b4c52000-feee-11e9-9b9a-84269187a539.png)

![Screenshot from 2019-11-04 10-10-03](https://user-images.githubusercontent.com/30416447/68100573-bb539780-feee-11e9-97de-3a0f82815f21.png)

**Comment by davidADSP** [comment]
Actually, I think I meant to remove this requirement as CycleGAN now uses `from tensorflow_addons.layers import InstanceNormalization`. 

instead of 

`from keras_contrib.layers.normalization import InstanceNormalization`

Can you just remove the requirement and see if that works?
